README.md

```diff
@@ -120,6 +120,11 @@ module.exports = async function() {
       dependencies will be restored to their prior state.
     */
     useYarn: true,
+    /*
+      If set to true, all npm scenarios will use `pnpm` for install with the `--frozen-lockfile=false` option. At cleanup, your
```

Comment:
Is `--frozen-lockfile` the same as ignoring the lockfile?

Reply:
not quite, but it'll allow things to float a bit and update the lockfile if package.json's change.
https://pnpm.io/cli/install#--frozen-lockfile

There isn't exactly a `--no-lockfile` option, like what yarn has, though there is a `--fix-lockfile` option: https://pnpm.io/cli/install#--fix-lockfile (docs are very light there)

I didn't have any problems with the currently implemented behavior, but something goofy I've noticed with pnpm in general is that when it resolves `^` or `~` ranges, it resolves first to what your have in your "store" (for speed) -- this can be somewhat annoying when I want "latest" -- but also, I heavily rely on automated dependency updating for meaningful dep updates.

Reply:
That may somewhat defeat the point of ember-try; I wonder if ember-try should by default put the specific version in `overrides` or `resolutions`.

Reply:
yeah, I think for consistent, it'd be cool to [`await latestVersion(range)`](https://www.npmjs.com/package/latest-version) on the try-config versions -- because at least then we can assure consistent behavior regardless of package manager. (anyone tried if yarn3/4 does anything goofy? :sweat_smile: )

Reply:
You could do that today in a try config, but if it isn't set in resolutions or overrides, you'll get whatever the package manager gives you for the version that is set.

Reply:
yeep